ScnSoft-Education/LeetCodeTraining/ZigzagConversion5/ZigzagConversion.cs

```diff
@@ -0,0 +1,37 @@
+﻿
+namespace LeetCodeTraining.ZigzagConversion5
+{
+    public class Solution
+    {
+        public static string Convert(string s, int numRows)
+        {
```

Comment:
check if 's' IsNullOrEmpty and add unit test for it

Reply:
adding IsNullOrEmpty